migrator/migrations/m_179_to_m_180_openshift_policy_exclusions/migration.go

```diff
@@ -0,0 +1,106 @@
+package m179tom180
+
+import (
+	"embed"
+
+	"github.com/pkg/errors"
+	"github.com/stackrox/rox/generated/storage"
+	"github.com/stackrox/rox/migrator/migrations"
+	postgresHelper "github.com/stackrox/rox/migrator/migrations/m_179_to_m_180_openshift_policy_exclusions/postgres"
+	"github.com/stackrox/rox/migrator/migrations/policymigrationhelper"
+	"github.com/stackrox/rox/migrator/types"
+	"github.com/stackrox/rox/pkg/postgres"
+)
+
+var (
+	migration = types.Migration{
+		StartingSeqNum: 179,
+		VersionAfter:   &storage.Version{SeqNum: 180},
+		Run: func(databases *types.Databases) error {
+			err := updatePolicies(databases.PostgresDB)
+			if err != nil {
+				return errors.Wrap(err, "updating policies")
+			}
+			return nil
+		},
+	}
+
+	//go:embed policies_before_and_after
+	policyDiffFS embed.FS
+
+	// We want to migrate only if the existing policy sections and title haven't changed.
```

Comment:
Ok so the comment is different from what is actually done. It says check policy sections and title, but in this case you are checking description.

I recommend, `policymigrationhelper.PolicySectionComparator`, `policymigrationhelper.DescriptionComparator`, and `policymigrationhelper.NameComparator`